src/modules/utils/motordrivercontrol/MotorDriverControl.h

```diff
@@ -38,20 +40,26 @@ class MotorDriverControl : public Module {
 
         void enable(bool on);
         int sendSPI(uint8_t *b, int cnt, uint8_t *r);
-
+        int sendUART(uint8_t *b, int cnt, uint8_t *r);
+        
         Pin spi_cs_pin;
         mbed::SPI *spi;
+        Pin sw_uart_tx_pin;
```

Comment:
this is a waste of valuable RAM. Why do we need to have BOTH SPI and UART RAM used when it can only use one or the other? We really do not have the luxury of wasting RAM like this.

Reply:
We can actually have both if we use a UART driver and also a SPI driver in the same printer for any reason. It would be better if we have defined any UART pins, so I could assign uart channels to UART pins as it's done for SPI drivers, however we are using soft uart and any board pins can be configured. Perhaps this should be configured inside the specific driver module instead of main MotorDriver module?